pages/in/widget/iscn/index.vue

```diff
@@ -287,6 +295,33 @@ export default {
     },
   },
   async mounted() {
+    const showDialogAction = !this.redirectUri;
```

Comment:
What is this `showDialogAction` for? It is not used in `calculateISCNTxTotalFee` seems